programs/clearing_house/src/math/margin.rs

```diff
@@ -492,6 +539,14 @@ pub fn meets_withdraw_margin_requirement(
         )?;
     }
 
+    if num_of_liabilities > 1 {
```

Comment:
dont you need to check this when people are opening/closing perps also?

Reply:
added in `meets_place_order_margin_requirement` wdyt?

Reply:
doesnt worst case base amount include open orders?